scripts/prepare_sector_network.py

```diff
@@ -2781,12 +2781,12 @@ def remove_carrier_related_components(n, carriers_to_drop):
         snakemake = mock_snakemake(
             "prepare_sector_network",
             simpl="",
-            clusters="19",
-            ll="c1.0",
-            opts="Co2L",
+            clusters="4",
+            ll="c1",
+            opts="Co2L-4H",
             planning_horizons="2030",
             sopts="72H",

```

Comment:
Great :D
In this harmonization, probably sopts should be 144H, https://github.com/pypsa-meets-earth/pypsa-earth/blob/c12ce1add1ffbaafba4ff92dd2c821d00a6d73d5/config.default.yaml#L46-L55
not sure if you wish to include in this PR

Reply:
Totally agree! Fixed. Thanks 🙂